dashboard/tasks_test.py

```diff
@@ -60,13 +59,7 @@ def test_batch_update_user_date_for_active_users(self, mocked_subtasks):
         for user in self.all_working_users:
             user.is_active = False
             user.save()
-        now = now_in_utc()
-        UserCacheRefreshTimeFactory.create(
-            user=self.user2,
-            enrollment=now,
-            certificate=now,
-            current_grade=now,
-        )
+        UserCacheRefreshTimeFactory.create(user=self.user2, unexpired=True)
```

Comment:
From `dashboard.tasks.batch_update_user_data`:

```python
        refresh_time_limit = now_in_utc() - timedelta(hours=MAX_HRS_MAIN_TASK)

        users_expired_cache = UserCacheRefreshTime.objects.filter(
            Q(enrollment__lt=refresh_time_limit) |
            Q(certificate__lt=refresh_time_limit) |
            Q(current_grade__lt=refresh_time_limit),
            user__is_active=True,
            user__profile__fake_user=False
        ).values_list('user', flat=True).distinct()
```

These refresh times are considered expired if they are older than some number of hours _before_ the current time. If the refresh time is being set to now in our test cases, they're intended to be unexpired.